LICENSE.adoc

```diff
@@ -678,7 +678,7 @@ src/third_party/httplib.h
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 cpp-httplib - A C++11 single-file header-only cross platform HTTP/HTTPS library.
-Copied from commit 6b9ffc8b of https://github.com/yhirose/cpp-httplib[cpp-httplib].
+Copied from commit 06bfa7e0 of https://github.com/yhirose/cpp-httplib[cpp-httplib].
```

Comment:
Looks like `httplib.*` are identical to those in the later release v0.9.1, so I suggest:

```suggestion
Copied from https://github.com/yhirose/cpp-httplib[cpp-httplib] v0.9.1.
```